src/app/chart/line/line.component.ts

```diff
@@ -217,6 +217,47 @@ export class LineComponent implements AfterViewInit {
     targetData.rangeFakeSeries = fakeSeries;
   }
 
+  /**
+   * sortSeriesData
+   * sorts seriesData by date
+   * @param { Array<TargetData> } seriesData:
+   **/
+  sortSeriesData(seriesData: Array<TargetData>): void {
+    const chartData = this.chart.data;
+    seriesData.sort((a: TargetData, b: TargetData) => {
+      const dateA = Date.parse(a.date);
+      const dateB = Date.parse(b.date);
+      if (dateA > dateB) {
+        return 1;
```

Comment:
my suggestion is: would you like to add a constant to this sorting options? like `ASCENDANT = 1 `, `DESCENDENT = -1` `UNORDERED = 0`

Reply:
I don't do this in any other sort (in this or in any other project), and I don't see other Typescript authors do it either.

Reply:
the principle comes from the book `clean code` this guy wrote. https://www.youtube.com/watch?v=Wibk0IfjfaI
I understand.